cluster/engine_test.go

```diff
@@ -34,6 +34,7 @@ var (
 		Driver:          "driver-test",
 		KernelVersion:   "1.2.3",
 		OperatingSystem: "golang",
+		OSType:          "linux",
```

Comment:
We don't really have any good test cases at the `api/handlers.go` level, as far as I can tell. Is there a specific place you hand in mind for this test?